tests/quantization/torchao_integration/test_torchao.py

```diff
@@ -349,41 +347,39 @@ def check_serialization_expected_output(self, device, expected_output):
         with tempfile.TemporaryDirectory() as tmpdirname:
             self.quantized_model.save_pretrained(tmpdirname, safe_serialization=False)
             loaded_quantized_model = AutoModelForCausalLM.from_pretrained(
-                self.model_name, torch_dtype=torch.bfloat16, device_map=device
+                tmpdirname, torch_dtype=torch.bfloat16, device_map=device
```

Comment:
will it be more meaningful?
```suggestion
                tmpdirname, torch_dtype="auto", device_map=device
```

Reply:
torchao int4_weight_only only supports bfloat16 for now, but for other quant types, we set torch dtype as "auto".